pkg/volume/csi/csi_mounter.go

```diff
@@ -411,7 +411,7 @@ func (c *csiMountMgr) applyFSGroup(fsType string, fsGroup *int64) error {
 // isDirMounted returns the !notMounted result from IsLikelyNotMountPoint check
 func isDirMounted(plug *csiPlugin, dir string) (bool, error) {
 	mounter := plug.host.GetMounter(plug.GetPluginName())
-	notMnt, err := mounter.IsLikelyNotMountPoint(dir)
+	notMnt, err := mounter.IsNotMountPoint(dir)
```

Comment:
@msau42 PTAL here. Wanna be sure this is what you wanted so that it works with local volume with bind mount.